README.md

```diff
@@ -64,10 +65,11 @@ To use with your Models add the `mixins` attribute to the definition object of y
 | option | type | description | required |
 | ------ | ---- | ----------- | -------- |
 |relations| [String] | relations which you want to delete together with current model | true |
+|deepDelete| [Boolean] | enable or disable the deep delete fonction. If activated, the CascadeDelete will be executed on the deleted related models as well (if they have the CascadeDelete mixin specified) | false |
```

Comment:
Sorry, I'm not native english speaker so I make mistakes sometimes... Thanks I will correct it :)